### DISCUSSION TITLE
What this error "net::err_cert_common_name_invalid"

### DISCUSSION THREAD

**Discussion by Hwangtaehun** [twbs/bootstrap#39948] [Q&A] [answered]
I use bootstrap to create a home page and I got a net::err_cert_common_name_invalid error in only bootstrap css and bootstrap js. Please tell me how to fix it.

**Comment by Mixxe73** [comment]
Аналогичная проблема(

**Comment by AldairMontano** [comment]
I have the same problem

**Comment by julien-deramond** [comment, marked as answer]
Hello there, this is probably the same temporary issue as https://github.com/twbs/bootstrap/issues/39947. I suppose that you're using the CDN version of Bootstrap with jsdelivr, isn't it?
In this case, please follow https://github.com/twbs/bootstrap/issues/39947#issuecomment-2089721026.

**Comment by shivanraptor** [comment]
This SSL problem would affect millions of websites, I hope this will be fixed ASAP.

**Comment by LyraHarring** [comment]
The error [NET: ERR_CERT_COMMON_NAME_INVALID](https://certera.com/kb/how-to-fix-err_cert_common_name_invalid-in-chrome-ssl-common-name-mismatch-error/) is commonly known as SSL common name mismatch error. This error arises from a misconfiguration of certificates on a server, and it can disrupt the secure connection between a user and a website. Beyond its technical origins, this error can also be triggered by various factors such as third-party extensions, security programs like antivirus and firewalls, and more. Different factors can lead to this error some are as follows: - 

1) Improper HTTPS Transition
2) Unrecognized Self-signed SSL
3) Antivirus Interference
4) Browser Extension Impact
5) Misconfigured Proxy settings

There are different ways to fix the error, you need to understand properly in detail, as I was also facing such an error and got a proper detailed guide for the solution. Hope it helps!
